packages/kbn-search-api-panels/components/preprocess_data.tsx

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+  EuiText,
+  EuiThemeProvider,
+  EuiTitle,
+  EuiToken,
+  EuiPanel,
+  EuiLink,
+  EuiFlexGrid,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+
+export const PreprocessDataPanel: React.FC = () => {
+  return (
+    <EuiThemeProvider colorMode="dark">
+      <EuiPanel paddingSize="xl">
+        <EuiFlexGrid direction="column" columns={2} gutterSize="xl">
+          <EuiFlexItem>
+            <EuiFlexGroup alignItems="flexStart" justifyContent="flexStart">
+              <EuiFlexItem grow={false}>
+                <EuiToken iconType="sparkles" size="l" color="#466B8D" />
```

Comment:
Is there a theme color we can use for `#466B8D`? we should avoid using hard-coded colors if we can.

cc: @JoseLuisGJ

Reply:
So after reviewing the alternatives, let's use `$euiColorVis1_behindText` for the background and the `colors.text` for the icon value
![CleanShot 2024-05-02 at 12 02 28@2x](https://github.com/elastic/kibana/assets/3108788/c3ce16e1-870f-4652-9f84-423106642bed)

Reply:
Updated.